oshmem/proc/proc.h

```diff
@@ -38,8 +38,8 @@ struct oshmem_group_t;
 #define OSHMEM_PE_INVALID   (-1)
 
 struct oshmem_proc_data_t {
```

Comment:
@igor-ivanov i did the change but i lost it before committing.

here is the comment
```c
/* This struct will be copied into the padding field of an ompi_proc_t
 * so the size of oshmem_proc_data_t must be less or equal than
 * OMPI_PROC_PADDING_SIZE */
struct oshmem_proc_data_t {
    char * transport_ids;
    int num_transports;
};
```